stable/gocd/README.md

```diff
@@ -349,6 +351,23 @@ The gocd service account can be associated with an existing role in the namespac
 helm install --namespace gocd --name gocd-app --set rbac.roleRef=ROLE_NAME stable/gocd
 ```
 
+#### Agent service account:
+
+Service account can be configured specifically for agents. This configuration also allows for the reuse of the top level service account that is used to configure the server pod. The various settings and their possible states are described below:
+
+| Parameter | Description | Default |
+| --------------------------------------------- | ----------------------------------------------------------------------------------- | ------------------- |
+| `agent.serviceAccount.reuseTopLevelServiceAccount`                   |  Specifies whether the top level service account (also used by the server) should be reused as the service account for gocd agents                                 | `nil`                        |
+| `agent.serviceAccount.name`                   |  If reuseTopLevelServiceAccount is false, this field specifies the name of an existing service account to be associated with gocd agents. By default (name field is empty), no service account is created for gocd agents | `nil`                        |
+
+Possible states:
+
+|State | Effect                                                                         |
+|------|--------------------------------------------------|
+|reuseTopLevelServiceAccount = false and name = empty|The default service account for the namespace will be used.|
```

Comment:
default => `default`, `default` is the name of the SA.

Reply:
ok, fixed.